From 9371cdeab2a24c07a0572c9fda36babc9f8c7209 Mon Sep 17 00:00:00 2001
From: David King <amigadave@amigadave.com>
Date: Sun, 31 May 2015 21:32:57 +0100
Subject: [PATCH] Fix crash when pressing a key to change directory

Use the correct signal handler for key-press-event on the directory
browser tree view.

https://bugzilla.redhat.com/show_bug.cgi?id=1226671
---
 data/browser.ui | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/data/browser.ui b/data/browser.ui
index 115c0ce..7ad1794 100644
--- a/data/browser.ui
+++ b/data/browser.ui
@@ -127,7 +127,7 @@
                                                 <property name="headers-visible">False</property>
                                                 <property name="visible">True</property>
                                                 <signal name="button-press-event" handler="on_directory_tree_button_press_event"/>
-                                                <signal name="key-press-event" handler="Browser_Tree_Node_Selected"/>
+                                                <signal name="key-press-event" handler="Browser_Tree_Key_Press"/>
                                                 <signal name="popup-menu" handler="on_directory_tree_popup_menu"/>
                                                 <signal name="row-collapsed" handler="collapse_cb" swapped="true"/>
                                                 <signal name="row-expanded" handler="expand_cb" swapped="true"/>
-- 
2.4.2

